components/nordigen/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "@pipedream/nordigen",
+  "version": "0.0.1",
+  "description": "Pipedream Nordigen Components",
+  "main": "nordigen.app.mjs",
+  "keywords": [
+    "pipedream",
+    "nordigen"
+  ],
+  "homepage": "https://pipedream.com/apps/nordigen",
+  "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+  "license": "MIT",
+  "dependencies": {
+    "axios": "^0.21.1",
+  },
```

```suggestion
    "axios": "^0.21.1"
  },
```
It seems that there is an extra comma here